src/iosource/pcap/Source.cc

```diff
@@ -56,15 +56,39 @@ void PcapSource::OpenLive()
 	char tmp_errbuf[PCAP_ERRBUF_SIZE];
 
 	// Determine interface if not specified.
-	if ( props.path.empty() )
-		props.path = pcap_lookupdev(tmp_errbuf);
-
 	if ( props.path.empty() )
 		{
-		safe_snprintf(errbuf, sizeof(errbuf),
-			 "pcap_lookupdev: %s", tmp_errbuf);
-		Error(errbuf);
-		return;
+		pcap_if_t* devs;
+		auto res = pcap_findalldevs(&devs, tmp_errbuf);
```

Comment:
Is there a reason you called `pcap_findalldevs` here twice? `res` isn't used by anything below.

Reply:
Nope: a relic of last second refactor, thanks.